grails-app/i18n/messages_et.properties

```diff
@@ -1,3 +1,19 @@
+# General
+general.bool.false = Ei
+general.bool.true = Jah
+general.btn.clearAll.label = Puhasta kõik
+general.btn.close = Sulge
+general.btn.download.label = Laadi alla
+general.btn.download.title = Alla laadimine
+general.btn.cancel = Laadi juurde
```

Comment:
I don't believe this is correct.

Reply:
It's probably as "correct" as "Laadi alla"